worldedit_commands/locale/worldedit.ru.tr

```diff
@@ -1,4 +1,4 @@
-# textdomain: worldedit
+# textdomain: worldedit_commands
```

Comment:
doesn't this file also need to be renamed?
not sure about the convention

Reply:
done